src/signal.rs

```diff
@@ -915,6 +918,13 @@ mod entry {
                     transmute(string_trampoline), Box::into_raw(f) as *mut _)
             }
         }
+        fn connect_changed<F: Fn(Entry) + 'static>(&self, f: F) -> u64 {
```

Comment:
But leaving a blank line between functions would be consistent with the rest of the module.